mod_deploy/controllers.py

```diff
@@ -78,65 +78,62 @@ def is_valid_signature(x_hub_signature, data, private_key):
 def deploy():
     """Deploy the GitHub request to the test platform."""
     from run import app
-    if request.method != 'POST':
```

Comment:
Why get rid of this check?

Reply:
Since `@github_request` checks for the same and hence it is redundant.

Reply:
Please resolve conversation if satisfied.